scripts/pre_commit_linter.py

```diff
@@ -717,9 +726,12 @@ def _check_comments(all_files):
 
                 if line.startswith('#') and not next_line.startswith('#'):
                     # Check that the comment ends with the proper punctuation.
-                    if (line[-1] not in
-                            ALLOWED_TERMINATING_PUNCTUATIONS) and (
-                                not any(word in line for word in EXCLUDED_PHRASES)):
+                    punctuation_is_allowed = line[-1] in (
+                        ALLOWED_TERMINATING_PUNCTUATIONS)
+                    word_is_present_in_excluded_phrases = not any(
```

Comment:
all_words_in_comment_are_valid (or similar), perhaps?

When reading the variable name you've picked, I'm left wondering "which word"?

Reply:
I don't think this line checks the validity of all words in a comment, exactly. How about `any_word_is_present_in_excluded_phrases`?

Reply:
I suggest s/any/some to avoid ambiguity (in some readings, "any" can mean "all" instead of "there exists") but otherwise your version sounds good to me. Thanks!

Reply:
Sorry, I did not quite get this. Are you suggesting to replace "any" with "some" in the variable name? Thanks!

Reply:
Done.